modules/log6x-input-spec-filter-audit-infrastructure.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * observability/logging/logging-6.0/log6x-clf.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="log6x-input-spec-filter-audit-infrastructure_{context}"]
+= Filtering the audit and infrastructure log inputs by source
+
+You can define the list of `audit` and `infrastructure` sources to collect the logs by using the `input` selector.
+
+.Procedure
+
+. Add a configuration to define the `audit` and `infrastructure` sources in the `ClusterLogForwarder` CR.
+
++
+The following example shows how to configure the `ClusterLogForwarder` CR to define `aduit` and `infrastructure` sources:
++
+.Example `ClusterLogForwarder` CR
++
+[source,yaml]
+----
+apiVersion: observability.openshift.io/v1
+kind: ClusterLogForwarder
+# ...
+spec:
+  inputs:
+    - name: mylogs1
+      infrastructure:
+        sources: # <1>
+          - node
```

Comment:
type: infrastructure is mandatory  for an input. can we include it?

Reply:
Comment: https://github.com/openshift/openshift-docs/pull/80384/files/352862728b9e57af9c4867fd0031c815121908ee#r1721933528